content/en/docs/reference/config/analysis/ist0167/index.md

```diff
@@ -0,0 +1,39 @@
+---
+title: IneffectivePolicy
+layout: analysis-message
+owner: istio/wg-user-experience-maintainers
+test: n/a
+---
+
+This message occurs when a policy applied in your Istio service mesh has no impact. This might be due to the
+policy's configuration not correctly targeting any workloads or namespaces in your service mesh.
```

Comment:
My first thought was to change `not correctly` to `incorrectly`.  Reading it a few more times, I'm wondering if we can remove `correctly`, so it's `not targeting`? My concern with the later is if there is something planned in the future where correctness might be checked. We use correctness at the end of the page, so going with my first thought.
```suggestion
policy's configuration incorrectly targeting any workloads or namespaces in your service mesh.
```

Reply:
Thanks for the suggestion. I prefer using `incorrectly` because there may be situations where users target workloads, but the method they use is incorrect. This message is intended to verify correctness.